target/ppc/translate/vsx-impl.c.inc

```diff
@@ -1546,6 +1546,48 @@ static bool trans_LXVKQ(DisasContext *ctx, arg_X_uim5 *a)
     return true;
 }
 
+static bool trans_XVTLSBB(DisasContext *ctx, arg_X_bf_xb *a)
+{
+    TCGv_i64 xb, all_true, all_false, zero, at_flag, af_flag, crbf;
+    int64_t mask;
+
+    REQUIRE_INSNS_FLAGS2(ctx, ISA310);
+    REQUIRE_VSX(ctx);
+
+    mask = dup_const(MO_8, 1);
+    zero = tcg_constant_i64(0);
+    at_flag = tcg_constant_i64(8); /* 0b1000 */
+    af_flag = tcg_constant_i64(2); /* 0b0010 */
+
+    xb = tcg_temp_new_i64();
+    all_true = tcg_const_i64(mask);
+    all_false = tcg_const_i64(mask);
+    crbf = tcg_const_i64(0);
+
+    for (int dw = 0; dw < 2; dw++) {
+        get_cpu_vsr(xb, a->xb, dw);
+        tcg_gen_and_i64(all_true, all_true, xb);
+        tcg_gen_andc_i64(all_false, all_false, xb);
+    }
+
+    tcg_gen_movcond_i64(TCG_COND_EQ, all_true, all_true,
+                        tcg_constant_i64(mask), at_flag, zero);
```

Comment:
I'm thinking on how to get rid of this `tcg_constant_i64` here and on the function below.

One idea I had was to change `mask` to a `tcg_constant_i64(dup_const...)` and change `all_true` and `all_false` initializations to `tcg_const_i64(dup_const(MO_8, 1))`

Any better suggestion?

Reply:
This is done on the revised code I just sent.